velox/functions/prestosql/aggregates/SumAggregate.h

```diff
@@ -227,7 +202,7 @@ bool registerSumAggregate(const std::string& name) {
           .integerVariable("a_precision")
           .integerVariable("a_scale")
           .argumentType("DECIMAL(a_precision, a_scale)")
-          .intermediateType("DECIMAL(38, a_scale)")
+          .intermediateType("VARBINARY")
```

Comment:
Is this compatible with Presto?

Reply:
@Yuhta it should be. See https://github.com/facebookincubator/velox/issues/3751
I am building a Prestissimo cluster with this change to verify as well. I will keep you posted here.

Reply:
@Yuhta The sum aggregate works via Prestissimo now.
```
presto> select sum(ss_sales_price) from hive.test.dec;
 _col0  
--------
 0.6848 
(1 row)
```

Reply:
Looks like presto intermediates are varbinary. https://code.byted.org/dp/prestissimo/blob/bd_main/presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/LongDecimalWithOverflowStateSerializer.java#L34